Source/WebCore/svg/SVGAnimationElement.cpp

```diff
@@ -171,7 +171,7 @@ void SVGAnimationElement::parseAttribute(const QualifiedName& name, const AtomSt
         // http://www.w3.org/TR/SVG11/animate.html#ValuesAttribute
         m_values.clear();
         value.string().split(';', [this](StringView innerValue) {
-            m_values.append(innerValue.stripWhiteSpace().toString());
+            m_values.append(stripLeadingAndTrailingHTMLSpaces(innerValue.toString()));
```

Comment:
Even here it would be better to strip the spaces before making the `String` so we want something like this:

`m_values.append(innerValue.stripLeadingAndTrailingMatchedCharacters(isHTMLSpace<UChar>).toString());`